erpnext/docs/user/manual/en/buying/request-for-quotation.md

```diff
@@ -6,9 +6,25 @@ In ERPNext, You can create request for quotation directly by going to:
 
 Create Request for Quotation:
 
+For User
+
 ![Request For Quotation]({{docs_base_url}}/assets/img/buying/request-for-quotation.png)
 
 Create Supplier Quotation from Request for Quotation form:
 
 ![Request For Quotation]({{docs_base_url}}/assets/img/buying/make-supplier-quotation-from-rfq.png)
+
+
+For supplier
+
+Email request for quotation link to the supplier through system, supplier will login to the portal using his credentials.
```

Comment:
Improvement:
On receiving the email requesting a quotation, the supplier will need to click on the link present in the email. They will be directed to the login page, where they will need to enter their username and password, which they received when they were sent a Request for Quotation for the first time.